### PR TITLE
Remap the operator encoding for binary 0xd

### PR DESCRIPTION
**Pull request by binji**

### CODE REVIEW

File: src/wasm-common.h [hunk 1]
```diff
@@ -179,177 +179,177 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
  *  ============================ */
 #define WASM_FOREACH_OPCODE(V)                                          \
   V(___, ___, ___, 0, 0x00, UNREACHABLE, "unreachable")                 \
-  V(___, ___, ___, 0, 0x01, BLOCK, "block")                             \
```
Comment (KarlSchimpf) [inline src/wasm-common.h]: It was my understanding that NOP had opcode 0x01 (but I could be wrong).

Comment (KarlSchimpf) [inline src/wasm-common.h]: Oops, my bad. You did it right. I was looking at the deletions.

File: src/wasm-common.h [hunk 2]
```diff
@@ -179,177 +179,177 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
...
+  V(___, I32, I32, 2, 0x3b, I32_STORE16, "i32.store16")                 \
+  V(___, I32, I64, 1, 0x3c, I64_STORE8, "i64.store8")                   \
+  V(___, I32, I64, 2, 0x3d, I64_STORE16, "i64.store16")                 \
+  V(___, I32, I64, 4, 0x3e, I64_STORE32, "i64.store32")                 \
+  V(I32, ___, ___, 0, 0x3f, CURRENT_MEMORY, "current_memory")           \
+  V(I32, I32, ___, 0, 0x40, GROW_MEMORY, "grow_memory")                 \
+  V(I32, ___, ___, 0, 0x41, I32_CONST, "i32.const")                     \
+  V(I64, ___, ___, 0, 0x42, I64_CONST, "i64.const")                     \
+  V(F64, ___, ___, 0, 0x43, F64_CONST, "f64.const")                     \
+  V(F32, ___, ___, 0, 0x44, F32_CONST, "f32.const")                     \
```
Comment (dschuff) [inline src/wasm-common.h]: I think `F32_CONST` and `F64_CONST` are backwards.

Comment (binji) [inline src/wasm-common.h]: Oops, thanks.

**Comment by binji** [comment]
@KarlSchimpf The last failure was due to the GH DNS outage last friday :-|

**Pull request merged**